### PR TITLE
BUG add sysroot symlinks

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@isuruf this is ready for review!

File: recipe/install_ld.sh [hunk 1]
```diff
@@ -7,5 +7,11 @@ cd build
...
 
 CHOST="${ctng_cpu_arch}-${ctng_vendor}-linux-gnu"
+NEW_CHOST="${ctng_cpu_arch}-conda-linux-gnu"
 mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/$NEW_CHOST/bin
+mkdir -p $PREFIX/$CHOST/bin
 cp $PWD/install/$PREFIX/bin/$CHOST-ld $PREFIX/bin/$CHOST-ld
+ln -s $PREFIX/bin/$CHOST-ld $PREFIX/bin/$NEW_CHOST-ld
+ln -s $PREFIX/bin/$CHOST-ld $PREFIX/$NEW_CHOST/bin/ld
+ln -s $PREFIX/bin/$CHOST-ld $PREFIX/$CHOST/bin/ld
```
Comment (isuruf) [inline recipe/install_ld.sh:17]: These 2 lines are unneeded

File: recipe/install_binutils.sh [hunk 2]
```diff
@@ -6,8 +6,17 @@ cd build
...
 make install-strip
 export HOST="${ctng_cpu_arch}-${ctng_vendor}-linux-gnu"
+export NEW_HOST="${ctng_cpu_arch}-conda-linux-gnu"
+mkdir -p $PREFIX/$NEW_HOST/bin
 # Remove hardlinks and replace them by softlinks
-for tool in addr2line ar as c++filt dwp elfedit gprof ld ld.bfd ld.gold nm objcopy objdump ranlib readelf size strings strip; do
+for tool in addr2line ar as c++filt dwp elfedit gprof ld.bfd ld.gold nm objcopy objdump ranlib readelf size strings strip; do
   rm -rf $PREFIX/$HOST/bin/$tool
   ln -s $PREFIX/bin/$HOST-$tool $PREFIX/$HOST/bin/$tool || true;
+  ln -s $PREFIX/bin/$HOST-$tool $PREFIX/$NEW_HOST/bin/$tool || true;
```
Comment (isuruf) [inline recipe/install_binutils.sh:15]: This line is unneeded

File: recipe/install_binutils.sh [hunk 3]
```diff
@@ -6,8 +6,17 @@ cd build
 
 make install-strip
 export HOST="${ctng_cpu_arch}-${ctng_vendor}-linux-gnu"
+export NEW_HOST="${ctng_cpu_arch}-conda-linux-gnu"
+mkdir -p $PREFIX/$NEW_HOST/bin
 # Remove hardlinks and replace them by softlinks
-for tool in addr2line ar as c++filt dwp elfedit gprof ld ld.bfd ld.gold nm objcopy objdump ranlib readelf size strings strip; do
+for tool in addr2line ar as c++filt dwp elfedit gprof ld.bfd ld.gold nm objcopy objdump ranlib readelf size strings strip; do
```
Comment (isuruf) [inline recipe/install_binutils.sh:12]: Keep `ld` here

**Comment by beckermr** [comment]
OK. ready for another look @isuruf!

File: recipe/install_ld.sh [hunk 4]
```diff
@@ -7,5 +7,9 @@ cd build
 make DESTDIR=$PWD/install install-strip
 
 CHOST="${ctng_cpu_arch}-${ctng_vendor}-linux-gnu"
+NEW_CHOST="${ctng_cpu_arch}-conda-linux-gnu"
 mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/$NEW_CHOST/bin
+mkdir -p $PREFIX/$CHOST/bin
```
Comment (isuruf) [inline recipe/install_ld.sh:13]:
```suggestion
```

**Comment by beckermr** [comment]
OK. Shall we merge?

**Pull request merged**